docs/get-operation-must-not-be-long-running.md

```diff
@@ -0,0 +1,71 @@
+# GetOperationMustNotBeLongRunning
+
+## Category
+
+ARM Error
+
+## Applies to
+
+ARM OpenAPI(swagger) specs
+
+## Related ARM Guideline Code
+
+- RPC-Get-V1-14
+
+## Output Message
+
+The GET operation cannot be Long Running and it MUST NOT have `x-ms-long-running-operation-options` property block defined.
+
```

Comment:
The GET operation cannot be long running. It must not have the 'x-ms-long-running-operation' and `x-ms-long-running-operation-options` properties defined. It must not have the status codes other than 200 and default.

Reply:
I am ignoring status codes part as its already covered by RPC-Get-V1-01